btrack/core.py

```diff
@@ -667,4 +667,8 @@ def to_napari(
 
         ndim = self.configuration.volume.ndim if ndim is None else ndim
 
-        return utils.tracks_to_napari(self.tracks, ndim, replace_nan=replace_nan)
+        return utils.tracks_to_napari(self.tracks, ndim=ndim, replace_nan=replace_nan)
+
+
+if __name__ == "__main__":
```

Comment:
just wondering why you've added this?

Reply:
ahh - I knew I missed something when resolving the conflicts!